river/anomaly/zscore.py

```diff
@@ -0,0 +1,67 @@
+import math
+
+from river.stats.var import RollingVar
+
+from .base import AnomalyDetector
+
+
+class Zscore(AnomalyDetector):
+    """
+    Anomaly detection based on zscore. Data is assumed to have gaussian distribution
+
+    Examples
+    --------
+    import numppy as np
+    from river.anomaly.zscore import Zscore
+
+    #warmup anomaly detector
+    zs = Zscore(100)
+    mean = 10.0
+    sd = 2.0
+
+    for _ in range(110):
+        v = np.random.normal(mean, sd)
+        zs.learn_one(v)
+
+    #get score
+    v = mean + 4 * sd
+    sc = zs.score_one(v)
+    print(sc)
+
+    """
+
+    def __init__(self, window_size: int, zs_threshold: float = 3.0):
+        """
+
+        Parameters
+        ----------
+        window_size : size of rolling window
+        zs_threshold : zscore threshold
+        """
```

Comment:
By body doc string, if you mean class level doc string, I will move it. I found this format in  svm.py. it's also the format created by PyCharm IDE

Reply:
This doc string is from river/anomaly/svm.py. Let me know which module to use as proper example for format

"""One-class SVM for anomaly detection.

    This is a stochastic implementation of the one-class SVM algorithm, and will not exactly match
    its batch formulation.

    It is encouraged to scale the data upstream with `preprocessing.StandardScaler`, as well as use
    `feature_extraction.RBFSampler` to capture non-linearities.

    Parameters
    ----------
    nu
        An upper bound on the fraction of training errors and a lower bound of the fraction of
        support vectors. You can think of it as the expected fraction of anomalies.
    optimizer
        The sequential optimizer used for updating the weights.

Reply:
I'm sorry I don't understand. You simply have to use the same format we use all over River

Reply:
Checked few files in river for format. Parameter description is in the next line and indented. I can make that change. BTW, I found few python files without any doc string at all. I can make all the changes except for using zscore for multivariate data, for reasons given earlier. It will take a dict and ensure there is only one value and use that value for univariate anomaly detection. If you let me know this acceptable, I will create another pull request.

Reply:
> BTW, I found few python files without any doc string at all

Do you have an example? Those are probably not public files so it's ok. Our CI process checks the docstring format so normally all public stuff has proper docstrings.